resources/views/database/links.blade.php

```diff
@@ -6,34 +6,62 @@
+                <thead>
                 <tr>
-                    <th scope="row">{{$loop->index+1}}</th>
-                    <td>{{$link->linkText}}</td>
-                    <td>
-                        <form method="POST" action="{{ route('deleteLink') }}">
-                            {{ csrf_field() }}
-                            <input id="delete" name="delete" value="delete" hidden>
-                            <input id="linkId-{{$link->link_id}}" name="link_id" value="{{$link->link_id}}" hidden>
-                            <button type="submit" class="btn btn-danger">Delete</button>
-                        </form>
-                    </td>
-                    <td>
-                        <a href="{{route('modifyLink', ['link_id' => $link->link_id])}}" class="btn btn-primary"><i class="fas fa-edit" aria-hidden="true"></i> Modify </a>
-                    </td>
+                    <th scope="col">#</th>
+                    <th scope="col">Name</th>
+                    <th scope="col">Link Text</th>
+                    <th scope="col">Status</th>
+                    <th scope="col">Last Update</th>
+                    <th scope="col" colspan="2">Actions</th>
                 </tr>
-            @endforeach
-            </tbody>
-        </table>
+                </thead>
+                <tbody>
+                @foreach($links as $link)
+                    <tr>
+                        <th scope="row">{{$loop->index+1}}</th>
+                        <td>{{$link->name}}</td>
+                        <td>{{$link->linkText}}</td>
+                        <td>{{$link->status}}</td>
+                        <td>{{$link->updated_at}}</td>
+                        <td>
+                            <form method="POST" action="{{ route('deleteLink') }}">
+                                {{ csrf_field() }}
+                                <input id="delete" name="delete" value="delete" hidden>
+                                <input id="linkId-{{$link->link_id}}" name="link_id" value="{{$link->link_id}}" hidden>
+                                <button type="submit" class="btn btn-danger">Delete</button>
+                            </form>
+                        </td>
+                        <td>
+                            <a href="{{route('modifyLink', ['link_id' => $link->link_id])}}" class="btn btn-primary"><i class="fas fa-edit" aria-hidden="true"></i> Modify </a>
+                        </td>
+                    </tr>
+                @endforeach
+                </tbody>
+            </table>
+            <div class="pagination">
+                {{ $links->links() }}
+            </div>
+        </div>
     </div>
 
 @endsection
+
+@push('css')
+    <style>
+        .table th {
+            text-align: center;
+        }
+
+        .table td {
+            text-align: center;
+        }
+
+        .pagination {
+            justify-content: center;
+        }
+    </style>
+
+@endpush
```

Comment:
OK for now, but let's use the Bootstrap classes `text-center` and `d-flex justify-content-center` next time.

Reply:
Yeah I was stuck on getting the table headers & data centered so I just used css, how can I do this? 
I managed to bootstrap the pagination though with `d-flex justify-content-center`

Reply:
this is an example of doing it <h2 class="text-center"> name</h2>

Reply:
ty